stravalib/client.py

```diff
@@ -959,7 +960,7 @@ def delete_activity(self, activity_id: int) -> None:
 
     def get_activity_zones(
         self, activity_id: int
-    ) -> list[model.BaseActivityZone]:
+    ) -> List[model.BaseActivityZone]:
```

Comment:
Any reason to change that to upper case. We've been quite consistent with the lower case annotation (which is the newest way)

Reply:
@enadeau gosh i must understand this incorrectly. I've been told to use uppercase `List`, `Dict` etc from the typing module. But you're saying that lower is newest? i'm actually a bit confused about this because there is the `|` annotation vs `Union` and then `from future import` for older python versions. i'm really not sure what is best practice here. but if you are saying lowercase is best i can fix this as i implement it throughout!

Reply:
fixed! just discovered how good the mypy docs are. i understand now 3.9+ is now lowercase. 3.8 and below use typing module! i just pushed a fix.

Reply:
Ok thanks @enadeau i'll add the from `__future__ import annotations` and will replace union and optional. that is going to make the docstrings a bit messier! eeks! i also see a failure on 3.8 and am curious if it's because i'm missing that import...

Reply:
Yup. it's a typing issue. OK i'm learning a lot here - thank you!!